src/components/Panel/PanelBase.tsx

```diff
@@ -12,18 +10,8 @@ interface Props {
   closePanel: () => void;
 }
 export function PanelBase({ children, panelOpen, closePanel }: Props) {
-  const router = useRouter();
   const isClient = useIsClient();
 
-  useEffect(() => {
-    router.events.on("routeChangeStart", closePanel);
-
-    return () => {
-      router.events.off("routeChangeStart", closePanel);
-    };
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, []);
```

Comment:
this was causing infinite loop, things seem to work without though may need closer inspection